canonical-web-and-design/ubuntu.com
# update buy.u.c links to point to /advantage

## Done

Updated all buy.u.c to point to /advantage instead, and removed external link icons where necessary.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit the following pages and see that links to the shop lead to /advantage:
  - /containers
  - /desktop/organisations
  - /pricing
  - /pricing/infra
  - /security/esm
  - /security
  - /support
  - /support/thank-you
